lib/cfa/augeas_parser.rb

```diff
@@ -193,6 +201,13 @@ def save_to_augeas(aug, prefix)
       end
     end
 
+    def ==(other)
+      other.class == self.class &&
+        other.data == data
+    end
+
+    alias_method :eql?, :==
```

Comment:
I don't remember offhand what `eql?` is for and why this alias is needed/useful, I'd have to look it up. IMHO it would be useful to add a comment explaining *why* we do this.